sdk/attestation/azure-security-attestation/vcpkg/portfile.cmake

```diff
@@ -1,6 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+# NOTE: All changes made to this file may get overwritten by the next port release.
+# Please contribute your changes to https://github.com/Azure/azure-sdk-for-cpp.
```

Comment:
I would be more assertive and say "will".

Reply:
Thanks! Updated. `"All changes made to this file will get overwritten by the next port release."`